Source/BeWake.rdef

```diff
@@ -0,0 +1,70 @@
+	$"FFFFFFFF0000001B3F3F3F3F3F3F3F3F3F0000FFFFFFFFFF00FFFF000000FFFF"
+	$"FFFFFF001B3F1B3F3F1B3F1C1B161919193F3F0000FFFF000000FFFFFF00FFFF"
+	$"FFFF003F3F1B1C1B3F3F3F3F1C1B1C161619193F1C0000FFFFFFFFFF00FFFFFF"
+	$"FFFF003F153F3F1B3F1B1C1C3F3F1C161616161919193F00FFFFFF000000FFFF"
+	$"FFFF003F3F3F3F3F3F1B1C3F1C1C3F3F1C1B1C161619193F00FFFFFFFFFFFFFF"
+	$"FFFF003F3F3F3F3F3F3F3F3F1C1B1C1C3F3F3F1B1C16163F3F00FFFFFFFFFFFF"
+	$"FF003F3F3F3F153F3F3F3F3F3F1B1C3F1C1C3F3F3F16163F3F3F00FFFFFFFFFF"
+	$"FF003F3F3F3F1515153F3F3F3F3F3F3F1C3F1C1C1B1C3F3F3F3F3F00FFFFFFFF"
+	$"FF003F3F3F1517171715153F3F3F3F3F3F3F3F3F1C1C3F1C3F3F3F00FFFFFFFF"
+	$"FF003F3F3F15171717171715153F3F3F3F3F3F3F3F3F1C3F3F3F3F3F00FFFFFF"
+	$"FF003F3F3F1517173F3F17171715153F3F3F3F3F3F3F3F3F3F0E3F3F00FFFFFF"
+	$"003F3F3F3F15173F3F3F3F3F3F171715153F3F3F3F3F3F3F0E0F3F3F00FFFFFF"
+	$"003F3F3F3F17173F3F3F3F3F3F3F3F171715153F3F3F3F3F0F0E3F3F00FFFFFF"
+	$"003F0000000000003F3F3F3F3F3F3F3F1717173F3F3F113F3F0F0E3F3F00FFFF"
+	$"000000150A0A1C000000003F3F3F3F3F3F3F3F113F3F113F3F0F0F3F3F00FFFF"
+	$"FFFF00150A0A1C0000000000003F3F3F3F3F3F113F3F113F3F3F3F3F3F0000FF"
+	$"FFFF00150A0A1C00000000000000003F3F3F3F113F3F113F3F3F3F00000000FF"
+	$"FFFF00150A0B1C000000000000000000003F113F3F3F113F3F3F0000FFFFFFFF"
+	$"FFFF001500001B1B1C000000000000000000123F3F3F113F3F000000FFFFFFFF"
+	$"FFFF0000151515151B1C1C00000000000000123F3F3F113F3F000F00FFFFFFFF"
+	$"FFFFFF000015151515151C1C1C000000000000003F3F3F11000F0F001111FFFF"
+	$"FFFFFFFFFF0000151515151C1C1C1B000000000000003F3F000F0F00111111FF"
+	$"FFFFFFFFFFFFFF0000151515151B1C1C1B0000001B1500000F0F0F0011111111"
+	$"FFFFFFFFFFFFFFFFFF0000151515151C1B1C1C001B16150F0F0E0F0011111111"
+	$"FFFFFFFFFFFFFFFFFFFFFF0000151615151C1B1C1C15150F0F0F0F0011111111"
+	$"FFFFFFFFFFFFFFFFFFFFFFFFFF0000151615151C1B16150F0F0F0E00111111FF"
+	$"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFF00001515151615150F0F0F00111111FFFF"
+	$"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF0000151515160E0F00111111FFFFFF"
+	$"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF000016150F00111111FFFFFFFF"
+	$"FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF000000111111FFFFFFFFFF"
+};
+
+resource mini_icon array {
+	$"FFFFFFFF0404040404FFFFFF00FFFFFF"
+	$"FFFF00003F3F3F3F3F00FFFF00FF00FF"
+	$"FF003F1C3F3F1C1C16190A00FFFF00FF"
+	$"FF003F3F3F1C1C3F1C1C161900FFFFFF"
+	$"FF043F153F3F3F1C1C3F3F163F00FFFF"
+	$"043F3F1717153F3F3F3F1C3F3F0AFFFF"
+	$"043F3F173F1717153F3F3F3F3F3F00FF"
+	$"003F3F173F3F17173F153F3F0F3F00FF"
+	$"00000A0A00000017173F3F113F0F04FF"
+	$"FF000A1C00000000173F3F113F3F00FF"
+	$"FF00001C1C00000000123F113F00FFFF"
+	$"FF000016161C000000003F3F000611FF"
+	$"FFFFFF000016161C1B000A000F061111"
+	$"FFFFFFFFFF000016151B1C150F061111"
+	$"FFFFFFFFFFFFFF06001516150F0011FF"
+	$"FFFFFFFFFFFFFFFFFF0600060011FFFF"
+};
+
+resource app_signature "application/x-vnd.simbad-BeWake";
+
+resource app_version {
+	major  = 1,
+	middle = 1,
+	minor  = 0,
+
+	variety = B_APPV_DEVELOPMENT,
+	internal = 0,
+
+	short_info = "BeWake",
+	long_info = "BeWake screensaver module, v:1.1"
```

Comment:
Let's remove the version ", v:1.1" from the long_info. That's what the "app_version" above is for.